### PR TITLE
tardis-infinite-world to development

### PR DESCRIPTION
**Pull request by DaanWesterhof**
I would like to merge the infinite world for the tardis to the development branch

### CODE REVIEW

File: tardis-world/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+        <script src="https://unpkg.com/aframe-simple-sun-sky@^1.2.2/simple-sun-sky.js"></script>
+        <script tree.object3D.position.x += 1;></script>
```
Comment (DaanWesterhof) [inline tardis-world/index.html:6]: This line makes no sense

**Review by DaanWesterhof — COMMENTED** [review]
There is a weird script in index.html

**Review by AHipWhale — CHANGES_REQUESTED** [review]
Fix the index.html

**Review by AHipWhale — APPROVED** [review]
Looks good

**Pull request merged**